src/Containers/Microsoft.NET.Build.Containers/Tasks/CreateNewImage.cs

```diff
@@ -159,6 +177,13 @@ internal async Task<bool> ExecuteAsync(CancellationToken cancellationToken)
                 GeneratedArchiveOutputPath = archive.ArchiveOutputPath;
             }
         }
+        catch (ContainerHttpException e)
+        {
+            if (BuildEngine != null)
```

Comment:
What's this condition about and why isn't it needed for the other error types?

Reply:
This was a miss - this class is tested without a full engine/test harness and so it doesn't have the engine/logging infra. I'll add it here for consistency.